src/game.cpp

```diff
@@ -3271,6 +3255,12 @@ void Game::processClientEvents(CameraOrientation *cam, float *damage_flash)
 
 			delete(event.show_formspec.formspec);
 			delete(event.show_formspec.formname);
+		} else if (event.type == CE_SHOW_LOCAL_FORMSPEC) {
+			FormspecFormSource *fs_src = new FormspecFormSource(event.show_formspec_local.formspec);
+			std::string name = event.show_formspec_local.formname;
```

Comment:
use directly event.show... in LocalFormspecHandler, var creation is not required there